doc/en-us/dev/js/dev_faq.md

```diff
@@ -43,12 +43,11 @@ BPlugin.js
 import {A} from "APlugin.js";
 console.log(A);
 ```
-其中APlugin.js中的`console.log("this is A plugin");`会被执行两次，在加载APlugin时会输出一次，再加载BPlugin时，由于B插件导入了`APlugin`插件的A变量，这会在B插件的`Context`完全载入一次A插件，即又输出一次。
+where the APlugin.js `console.log("this is A plugin");` will be executed twice, in the load APlugin will output once, and then load BPlugin, because the B plug-in imported the `APlugin` plug-in A variable, this will be in the B plug-in `Context` completely loaded once A plug-in which is output once again.
 
-而其中的`onlyOnceExecute`来自[LLSE-Lib](https://github.com/PowerNukkitX/LLSE-Lib)库，就是为了解决该问题出现的。
-`onlyOnceExecute`第一个参数传入一个回调函数，第二个参数传入一个UUID作为唯一标识符，这个回调函数只会执行一次，不管多少个插件导入该插件。
-
-但是由于PNXJS的每个JS插件都是一个独立的`Context`，所以B插件导入了`APlugin`插件的A变量，A变量是没有初始化的，同时B插件的`Context`完全载入一次A插件，但是由于`onlyOnceExecute`全局只会执行一次，所以A变量在B插件的`Context`不会被初始化
+The `onlyOnceExecute` from the [LLSE-Lib](https://github.com/PowerNukkitX/LLSE-Lib) library is designed to solve this problem.
+The first parameter of `onlyOnceExecute` passes a callback function and the second parameter passes a UUID as a unique identifier, and this callback function will only be executed once, no matter how many plugins import the plugin.
 
+But since each JS plug-in of PNXJS is a separate `Context`, the B plug-in imports the A variable of the `APlugin` plug-in, which is not initialized, while the `Context` of the B plug-in loads the A plug-in completely once, but since `onlyOnceExecute` will only be executed once globally, the A variable in the B plugin's `Context` will not be initialized
```

Comment:
Please replace all `plug-in` with `plugin`.

Reply:
fix in next commit

Reply:
commit c726061604c73240c6e1dbc554a7eeb6fcfc7182